java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java

```diff
@@ -0,0 +1,94 @@
+package ro.siit.java8;
+
+/**
+ * @author Alin_Negrea
+ * A bubble  sort algorithm was implemented to sort an array of Sales Representative objects in descending order.
+ * The sorting was done based on the amount of revenue generated via sales.
+ * Each representative has a number of sale, and a quota/sale.
+ *
+ */
+public class Main {
```

Comment:
Java class files must have the same name as the class: Main.java and not main.java.